azure-spring-boot/src/main/java/com/microsoft/azure/spring/autoconfigure/aad/AADAuthenticationFilterAutoConfiguration.java

```diff
@@ -80,6 +89,12 @@ public ResourceRetriever getJWTResourceRetriever() {
                 aadAuthProps.getJwtSizeLimit());
     }
 
+    @Bean
```

Comment:
```suggestion
    @Bean
     @ConditionalOnMissingBean(JWKSetCache.class)
    public JWKSetCache getJWKSetCache() {
        return new DefaultJWKSetCache(properties.lifespan, TimeUnit.MILLISECONDS)
     }
```

Reply:
Would this be something like what we would want to do? We would then define the lifespan within a properties file

Reply:
Sound good! It is a cool way to customize the cache by properties. If users don't set related configuration, it will use 5 min by default.